src/uct/ib/mlx5/ib_mlx5.h

```diff
@@ -939,6 +938,11 @@ uct_ib_mlx5_devx_query_qp_peer_info(uct_ib_iface_t *iface, uct_ib_mlx5_qp_t *qp,
 
 static inline void uct_ib_mlx5_devx_destroy_qp(uct_ib_mlx5_md_t *md, uct_ib_mlx5_qp_t *qp) { }
 
+static inline uint8_t
+uct_ib_mlx5_devx_md_get_counter_set_id(uct_ib_mlx5_md_t *md, uint8_t port_num) {
```

Comment:
maybe use HAVE_DEVX inside uct_ib_mlx5_iface_get_counter_set_id?

Reply:
may we deside on it? here I followed [kernel style](https://github.com/torvalds/linux/blob/master/Documentation/process/coding-style.rst#21-conditional-compilation)

Reply:
that seems a good and justified approach, though we never followed it yet in the code
so can keep it so, just fix code style { on next line